datafusion/sqllogictest/test_files/information_schema.slt

```diff
@@ -136,91 +136,91 @@ statement ok
-datafusion.execution.aggregate.scalar_update_factor 10
-datafusion.execution.batch_size 8192
-datafusion.execution.coalesce_batches true
-datafusion.execution.collect_statistics false
-datafusion.execution.meta_fetch_concurrency 32
-datafusion.execution.parquet.allow_single_file_parallelism false
-datafusion.execution.parquet.bloom_filter_enabled false
-datafusion.execution.parquet.bloom_filter_fpp NULL
-datafusion.execution.parquet.bloom_filter_ndv NULL
-datafusion.execution.parquet.column_index_truncate_length NULL
-datafusion.execution.parquet.compression zstd(3)
-datafusion.execution.parquet.created_by datafusion
-datafusion.execution.parquet.data_page_row_count_limit 18446744073709551615
-datafusion.execution.parquet.data_pagesize_limit 1048576
-datafusion.execution.parquet.dictionary_enabled NULL
-datafusion.execution.parquet.dictionary_page_size_limit 1048576
-datafusion.execution.parquet.enable_page_index true
-datafusion.execution.parquet.encoding NULL
-datafusion.execution.parquet.max_row_group_size 1048576
-datafusion.execution.parquet.max_statistics_size NULL
-datafusion.execution.parquet.metadata_size_hint NULL
-datafusion.execution.parquet.pruning true
-datafusion.execution.parquet.pushdown_filters false
-datafusion.execution.parquet.reorder_filters false
-datafusion.execution.parquet.skip_metadata true
-datafusion.execution.parquet.statistics_enabled NULL
-datafusion.execution.parquet.write_batch_size 1024
-datafusion.execution.parquet.writer_version 1.0
-datafusion.execution.planning_concurrency 13
-datafusion.execution.sort_in_place_threshold_bytes 1048576
-datafusion.execution.sort_spill_reservation_bytes 10485760
-datafusion.execution.target_partitions 7
-datafusion.execution.time_zone +00:00
-datafusion.explain.logical_plan_only false
-datafusion.explain.physical_plan_only false
-datafusion.explain.show_statistics false
-datafusion.optimizer.allow_symmetric_joins_without_pruning true
-datafusion.optimizer.bounded_order_preserving_variants false
-datafusion.optimizer.enable_round_robin_repartition true
-datafusion.optimizer.enable_topk_aggregation true
-datafusion.optimizer.filter_null_join_keys false
-datafusion.optimizer.hash_join_single_partition_threshold 1048576
-datafusion.optimizer.max_passes 3
-datafusion.optimizer.prefer_hash_join true
-datafusion.optimizer.repartition_aggregations true
-datafusion.optimizer.repartition_file_min_size 10485760
-datafusion.optimizer.repartition_file_scans true
-datafusion.optimizer.repartition_joins true
-datafusion.optimizer.repartition_sorts true
-datafusion.optimizer.repartition_windows true
-datafusion.optimizer.skip_failed_rules false
-datafusion.optimizer.top_down_join_key_reordering true
-datafusion.sql_parser.dialect generic
-datafusion.sql_parser.enable_ident_normalization true
-datafusion.sql_parser.parse_float_as_decimal false
+datafusion.catalog.create_default_catalog_and_schema true Whether the default catalog and schema should be created automatically.
```

Comment:
My slight concern is the output now may be too wide, and we prob should also consider to split `SHOW ALL`

`SHOW ALL EXTENDED` or `SHOW ALL VERBOSE` will output the description whereas `SHOW ALL` can still show  just name and value